src/main/java/core/basesyntax/figures/Square.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.figures;
+
+public class Square extends Figure {
+
+    private final double side;
+
+    public Square(String color, double side) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double calculateArea() {
+        return Math.round((side * side) * 10.0 / 10.0);
```

Comment:
The Math.round function is being used incorrectly. It should take a double value and round it to the nearest whole number, but here you are multiplying and dividing by 10.0 which doesn't change the value. If you want to round the area to a certain number of decimal places, consider using `BigDecimal` for precision or simply return the product of side * side if rounding is not required.